### PR TITLE
Block .docker folder and .env file from being checked in

### PR DESCRIPTION
**Pull request by Choongkyu**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,5 @@ node_modules
 .netlify
 
 /.idea
+.docker
+.env*
```
Comment (benawad) [inline .gitignore]: this ignores .env.example

Comment (Choongkyu) [reply .gitignore]: thanks! I'll change this to `.env` only.

File: .env.dev [hunk 2]
```diff
@@ -0,0 +1,11 @@
+ACCESS_TOKEN_SECRET=1234
+REFRESH_TOKEN_SECRET=1234
+GITHUB_CLIENT_ID=a4055d16d4a87f1a0001
+TWITTER_API_KEY=6u9ngIEUcl60LuFp11z7FdKaY  
+TWITTER_SECRET_KEY=M5W7O0kryvploFGfnmoHAQ9qZohzrRzWgI7Lh637MzVm86Bwy9 
+TWITTER_BEARER_TOKEN=AAAAAAAAAAAAAAAAAAAAAKfONAEAAAAAHzAgyWa7sWcbmhbfXV11yTW5IuI%3D3Ir6ByjTAIbIKjDLkhoj54yqgRo6gmxYAMV7o1H5GIOy6eNqxR
+GITHUB_CLIENT_SECRET=95d94c993c20faa462227623508296348e3df7e9
+SENTRY_DSN=https://3c6781dcfaa54868a2a1bc72428e2594@o528947.ingest.sentry.io/5646659
```
Comment (benawad) [inline .env.dev]: what sentry is this pointing to?

Comment (Choongkyu) [reply .env.dev]: I accidentally checked in all my credentials, which happened by un-wildcarding `.env`. haha. time to nuke them and remove this too. Thanks!

**Pull request merged**